modules/core/src/test/java/org/apache/ignite/internal/client/thin/ServiceAwarenessTest.java

```diff
@@ -90,11 +98,18 @@ public class ServiceAwarenessTest extends AbstractThinClientTest {
     /** */
     private static ListeningTestLogger clientLogLsnr;
 
+    /** */
+    private final UUID[] nodeIds = new UUID[BASE_NODES_CNT + TOP_UPD_NODES_CNT];
+
     /** {@inheritDoc} */
     @Override protected IgniteConfiguration getConfiguration(String igniteInstanceName) throws Exception {
         IgniteConfiguration cfg = super.getConfiguration(igniteInstanceName);
 
+        int nodeIdx = getTestIgniteInstanceIndex(igniteInstanceName);
+
         cfg.setDiscoverySpi(new TestBlockingDiscoverySpi());
+        cfg.setNodeId(nodeIds[nodeIdx]);
```

Comment:
This method is deprecated. Can we use  consistentId? Might be a map cstId -> UUID

Reply:
Done.